### PR TITLE
DLS-5259 Fix colour contrast issues

### PR DESCRIPTION
**Pull request by adamliptrot-oc**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -235,7 +235,7 @@ manage-details{
 }
 
 assets {
-  version = "3.11.0"
+  version = "3.25.0"
```
Comment (LAJW) [inline conf/application.conf:238]: ⛏️ This also doesn't seem to have anything to do with color contrast.

**Review by LAJW — APPROVED** [review]
⛏️ I'd add that we're bumping frontend assets and changing padding in the PR title/commit description (and separate into a separate commit next time).

File: conf/application.conf (hunk 1 above)
Comment (adamliptrot-oc) [reply conf/application.conf:238]: This changes the library to a version which includes the colour changes which were made in the move from GOVUK Elements to GOVUK Frontend.

File: app/assets/stylesheets/components/box-highlight.scss [hunk 2]
```diff
@@ -1,8 +1,8 @@
 .govuk-box-highlight {
   margin: 1em 0;
-  padding: 2em 0 1em;
+  padding: 2em 1em 1em;
```
[the remark this answers is not in the document]
Comment (adamliptrot-oc) [reply app/assets/stylesheets/components/box-highlight.scss:3]: Nope, but it fixes a minor layout issue and brings the visuals more inline with current patterns.

[the remark this answers is not in the document]
Comment (LAJW) [reply app/assets/stylesheets/components/box-highlight.scss:3]: Sure, could be in a separate commit though.

File: conf/application.conf (hunk 1 above)
Comment (LAJW) [reply conf/application.conf:238]: I would be nitpicking if this was included in the commit message :)

**Comment by adamliptrot-oc** [comment]
> I'd add that we're bumping frontend assets and changing padding in the PR title/commit description (and separate into a separate commit next time).

This information is in the commit message and the JIRA ticket.

**Comment by LAJW** [comment]
It's not in the commit message. I'm not blocking, just pointing it out :)
image

**Pull request merged**